taxi/forms.py

```diff
@@ -0,0 +1,43 @@
+from django import forms
+import re
+from taxi.models import Driver, Car
+
+
+class DriverForm(forms.ModelForm):
+    class Meta:
+        model = Driver
+        fields = "__all__"
+
+
+class DriverLicenseUpdateForm(forms.ModelForm):
+    class Meta:
+        model = Driver
+        fields = ("license_number",)
+
+    def clean_license_number(self):
+        license_number = self.cleaned_data["license_number"]
+
+        if len(license_number) != 8:
+            raise forms.ValidationError(
+                "The license number must contain 8 characters."
+            )
+
+        if not re.match(r"^[A-Z]{3}\d{5}$", license_number):
+            raise forms.ValidationError(
+                "The license number must consist of 3 uppercase letters "
+                "followed by 5 digits."
+            )
+
+        return license_number
+
+
+class CarForm(forms.ModelForm):
+    drivers = forms.ModelMultipleChoiceField(
+        queryset=Driver.objects.all(),
```

Comment:
The queryset for the 'drivers' field in the CarForm is set to 'Driver.objects.all()'. This will be evaluated when the server starts and will not reflect changes in the Driver model unless the server is restarted. Consider using a callable for the queryset to ensure it is always up-to-date.